apps/entity-service/src/entities/dto/site.dto.ts

```diff
@@ -55,6 +54,12 @@ export class SiteLightDto extends EntityDto {
   @ApiProperty({ nullable: true })
   name: string | null;
 
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project name"
+  })
+  projectName: string | null;
+
```

Comment:
There are lots of fields on the spreadsheet that aren't represented in this lite DTO, and projectName I didn't think was needed except in the full DTO. Is it actually needed here?

Reply:
I think I wrote it, but yes, we need this name on the index endpoint.